zipkin-server/pom.xml

```diff
@@ -129,19 +129,6 @@
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-log4j2</artifactId>
       <version>${spring-boot.version}</version>
-      <exclusions>
```

Comment:
@codefromthecrypt I think this was an issue I failed to troubleshoot properly (and derailed me towards SLF4J 1.7.x downgrade), sorry about that

Reply:
no biggie